boilerplate/scripts/build.sh

```diff
@@ -160,7 +160,7 @@ pushd $BOILERPLATE_DIR > /dev/null
 
 # Build the zip!
 # the node-X segment in the next line should match the latest major version
-zip -R $TARGET_FILE '*.js' '*.cjs' '*.json' '*/linux-x64-node-10/*.node' '*/linux-x64-node-12/*.node' '*/linux-x64-node-14/*.node'
+zip -R $TARGET_FILE '*.js' '*.cjs' '*.json' '*/linux-x64-node-10/*.node' '*/linux-x64-node-12/*.node' '*/linux-x64-node-14/*.node' '*/linux-x64-node-16/*.node'
```

Comment:
@eliangcs would there be any negative effects from removing these in regards to interacting with the monolith?

Reply:
@robwa10 those were required when legacy-scripting-runner still used deasync. Now those binary files don't matter anymore since legacy-scripting-runner [started using synckit](https://github.com/zapier/zapier-platform/pull/509). We can even remove all the `*.node` files from the line!